app/models/capital_summary.rb

```diff
@@ -1,16 +1,10 @@
 class CapitalSummary < ApplicationRecord
   belongs_to :assessment
-
-  has_many :properties, dependent: :destroy
-  has_many :additional_properties, -> { additional }, inverse_of: :capital_summary, class_name: "Property", dependent: :destroy
-  has_one :main_home, -> { main_home }, inverse_of: :capital_summary, class_name: "Property", dependent: :destroy
   has_many :eligibilities,
            class_name: "Eligibility::Capital",
            foreign_key: :parent_id,
            inverse_of: :capital_summary,
            dependent: :destroy
-  has_many :disputed_capital_items, -> { disputed }, class_name: "CapitalItem"
```

Comment:
I'm guessing these two `has_many` lines are being cleared away because they're redundant and unused? They don't seem to be directly related to properties, but it's fair enough if this is just some spring cleaning as we go.

Reply:
Good Catch. I will do cleanup in separate ticket.